feature/fortune/src/main/java/org/sopt/official/feature/fortune/feature/fortuneDetail/FortuneDetailScreen.kt

```diff
@@ -36,31 +36,46 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import org.sopt.official.designsystem.SoptTheme
+import org.sopt.official.feature.fortune.feature.fortuneDetail.component.PokeRecommendationDashboard
 import org.sopt.official.feature.fortune.feature.fortuneDetail.component.TodayFortuneDashboard
 import org.sopt.official.feature.fortune.feature.fortuneDetail.model.FortuneDetailUiState
 import org.sopt.official.feature.fortune.feature.fortuneDetail.model.FortuneDetailUiState.Error
 import org.sopt.official.feature.fortune.feature.fortuneDetail.model.FortuneDetailUiState.Loading
-import org.sopt.official.feature.fortune.feature.fortuneDetail.model.FortuneDetailUiState.TodaySentence
+import org.sopt.official.feature.fortune.feature.fortuneDetail.model.FortuneDetailUiState.Success
+import org.sopt.official.feature.fortune.feature.fortuneDetail.model.FortuneDetailUiState.Success.TodaySentence
+import org.sopt.official.feature.fortune.feature.fortuneDetail.model.FortuneDetailUiState.Success.UserInfo
 import timber.log.Timber
 
 @Composable
 internal fun FortuneDetailScreen(
     paddingValue: PaddingValues,
     date: String,
     onFortuneAmuletClick: () -> Unit,
+    onPokeClick: (userId: Long) -> Unit,
+    onProfileClick: (userId: Long) -> Unit,
     modifier: Modifier = Modifier,
     uiState: FortuneDetailUiState = Loading,
 ) {
     Column(
         horizontalAlignment = Alignment.CenterHorizontally,
-        modifier = modifier.fillMaxSize().padding(paddingValues = paddingValue),
+        modifier = modifier
+            .fillMaxSize()
+            .padding(paddingValues = paddingValue),
     ) {
         Spacer(modifier = Modifier.height(height = 16.dp))
         when (uiState) {
-            is TodaySentence -> {
+            is Success -> {
                 TodayFortuneDashboard(
                     date = date,
-                    todaySentence = uiState.message,
+                    todaySentence = uiState.todaySentence.message,
+                )
+                Spacer(modifier = Modifier.height(height = 20.dp))
+                PokeRecommendationDashboard(
+                    profile = uiState.userInfo.profile,
+                    name = uiState.userInfo.userName,
+                    userDescription = uiState.userInfo.userDescription,
+                    onPokeClick = { onPokeClick(uiState.userInfo.userId) },
+                    onProfileClick = { onProfileClick(uiState.userInfo.userId) }
```

Comment:
onPokeClick 함수에서 userId 값을 넘겨준다면 아래처럼 간단하게 쓸 수 있어요. 
아래처럼 작성한다면 코드가 간단해지고, 캡쳐되기 때문에 약간의 성능 향상이 있지만, id라는 값을 넘겨주는 것 때문에 너무 강한 결합이 생기지 않을까?? 라는 고민을 요즘 하고있습니다.  혹시 어떻게 생각하시나용???

```suggestion
                    onPokeClick = onPokeClick,
                    onProfileClick = onProfileClick
```

Reply:
오호~ 킹치만 호이스팅된 onXXXClick 함수들은 인자로 아무것도 넘겨주지 않아서 아래코드처럼 쓸 수 없습니당!
본 함수 람다에서 무엇을 넘겨줘야할지 구현해줘야해서 위처럼 작성했습니다 

실제 onPokeClick, onProfileClick 등을 생성자로 받고있는 하위 컴포넌트들은 클릭 시 id를 넘겨줘야함!과 같은 구체적인 구현사항에 의존하지 않고, 호이스팅 된 상위 컴포넌트에서 위처럼 람다를 구현해서 상위로 전달해줍니다.
람다와 id가 강한결합.. 사실 그렇게 와닿지 않아서 잘 이해가 가진 않아요! 결국 필연적인 로직이라 생각하거든요 :)

Reply:
아 코드를 제가 잘못 이해했었네요.
`PokeRecommendationDashboard`에 userInfo의 id도 같이 넘겨주고 있는 것으로 착각했습니다!